app/assets/javascripts/app.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Created by cuthbertm on 3/21/16.
+ */
+toastr.options = {
```

Comment:
I [used toastr for MapRoulette 1](https://github.com/osmlab/maproulette/blob/dd0c0bc3aa2dd2d4bdf86ebd70f1f84cd6271b26/jsx/maproulette.js#L16) because it was the most lightweight thing I could find that does what I needed it to (stylable, supports html content, variable auto dismiss time, sticky option, perhaps a few other minor things). I don't see a reason to change toast message frameworks but just curious if you had looked at any others?